jetty-xml/src/main/java/org/eclipse/jetty/xml/XmlConfiguration.java

```diff
@@ -332,14 +377,22 @@ public void initializeDefaults(Object object)
 
     private static class JettyXmlConfiguration implements ConfigurationProcessor
     {
-        private String _url;
+        private String _location;
         XmlParser.Node _root;
         XmlConfiguration _configuration;
 
+        @Override
+        public void init(Resource resource, XmlParser.Node root, XmlConfiguration configuration)
+        {
+            _location = resource == null ? "UNKNOWN_RESOURCE" : resource.toString();
+            _root = root;
+            _configuration = configuration;
+        }
```

Comment:
Also, this new `init()` method is a copy/paste of the old one. See above, implement one in terms of the other.

Reply:
I have removed the URL version.